Can't import FXML files with `<?language javascript?>`

<!--- Provide a brief summary of the issue -->

Since JavaFX 20, importing an FXML file with `<?language javascript?>` fails with:
```
java.io.IOException: javafx.fxml.LoadException: JavaScript script engine is disabled.
/hello-script.fxml

	at com.gluonhq.scenebuilder.kit@24.0.0-SNAPSHOT/com.oracle.javafx.scenebuilder.kit.fxom.FXOMLoader.handleUnknownAndMissingCauses(FXOMLoader.java:119)
	at com.gluonhq.scenebuilder.kit@24.0.0-SNAPSHOT/com.oracle.javafx.scenebuilder.kit.fxom.FXOMLoader.handleFxmlLoadingError(FXOMLoader.java:114)
	at com.gluonhq.scenebuilder.kit@24.0.0-SNAPSHOT/com.oracle.javafx.scenebuilder.kit.fxom.FXOMLoader.load(FXOMLoader.java:106)
	at com.gluonhq.scenebuilder.kit@24.0.0-SNAPSHOT/com.oracle.javafx.scenebuilder.kit.fxom.FXOMDocument.<init>(FXOMDocument.java:119)
	at com.gluonhq.scenebuilder.kit@24.0.0-SNAPSHOT/com.oracle.javafx.scenebuilder.kit.editor.EditorController.updateFxomDocument(EditorController.java:2520)
	at com.gluonhq.scenebuilder.kit@24.0.0-SNAPSHOT/com.oracle.javafx.scenebuilder.kit.editor.EditorController.setFxmlTextAndLocation(EditorController.java:713)
	at com.gluonhq.scenebuilder.app@24.0.0-SNAPSHOT/com.oracle.javafx.scenebuilder.app.DocumentWindowController.loadFromFile(DocumentWindowController.java:405)
	at com.gluonhq.scenebuilder.app@24.0.0-SNAPSHOT/com.oracle.javafx.scenebuilder.app.SceneBuilderApp.performOpenFiles(SceneBuilderApp.java:739)
	at com.gluonhq.scenebuilder.app@24.0.0-SNAPSHOT/com.oracle.javafx.scenebuilder.app.SceneBuilderApp.handleOpenFilesAction(SceneBuilderApp.java:538)
	at com.gluonhq.scenebuilder.app@24.0.0-SNAPSHOT/com.oracle.javafx.scenebuilder.app.welcomedialog.WelcomeDialogWindowController.openFilesAndHideStage(WelcomeDialogWindowController.java:315)
	at com.gluonhq.scenebuilder.app@24.0.0-SNAPSHOT/com.oracle.javafx.scenebuilder.app.welcomedialog.WelcomeDialogWindowController.attemptOpenExistingFiles(WelcomeDialogWindowController.java:308)
	at com.gluonhq.scenebuilder.app@24.0.0-SNAPSHOT/com.oracle.javafx.scenebuilder.app.welcomedialog.WelcomeDialogWindowController.handleOpen(WelcomeDialogWindowController.java:350)
	at com.gluonhq.scenebuilder.app@24.0.0-SNAPSHOT/com.oracle.javafx.scenebuilder.app.welcomedialog.WelcomeDialogWindowController.handleOpen(WelcomeDialogWindowController.java:291)
	at com.gluonhq.scenebuilder.app@24.0.0-SNAPSHOT/com.oracle.javafx.scenebuilder.app.welcomedialog.WelcomeDialogWindowController.fireOpenRecentProject(WelcomeDialogWindowController.java:239)
	at com.gluonhq.scenebuilder.app@24.0.0-SNAPSHOT/com.oracle.javafx.scenebuilder.app.welcomedialog.WelcomeDialogWindowController.lambda$loadAndPopulateRecentItemsInBackground$6(WelcomeDialogWindowController.java:200)
	at javafx.base@23.0.1/com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent(CompositeEventHandler.java:86)
	at javafx.base@23.0.1/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:232)
	at javafx.base@23.0.1/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:189)
	at javafx.base@23.0.1/com.sun.javafx.event.CompositeEventDispatcher.dispatchBubblingEvent(CompositeEventDispatcher.java:59)
	at javafx.base@23.0.1/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:58)
...
Caused by: javafx.fxml.LoadException: JavaScript script engine is disabled.
/hello-script.fxml

	at javafx.fxml@23.0.1/javafx.fxml.FXMLLoader.constructLoadException(FXMLLoader.java:2723)
	at javafx.fxml@23.0.1/javafx.fxml.FXMLLoader.processLanguage(FXMLLoader.java:2810)
	at javafx.fxml@23.0.1/javafx.fxml.FXMLLoader.processProcessingInstruction(FXMLLoader.java:2776)
	at javafx.fxml@23.0.1/javafx.fxml.FXMLLoader.loadImpl(FXMLLoader.java:2644)
	at javafx.fxml@23.0.1/javafx.fxml.FXMLLoader.load(FXMLLoader.java:2552)
	at com.gluonhq.scenebuilder.kit@24.0.0-SNAPSHOT/com.oracle.javafx.scenebuilder.kit.fxom.FXOMLoader.load(FXOMLoader.java:104)
	... 67 more

```
As per https://openjfx.io/javadoc/23/javafx.fxml/javafx/fxml/doc-files/introduction_to_fxml.html#scripting
Scene Builder needs to run with 
`-Djavafx.allowjs=true`

The same applies to the test that was modified in #619:

https://github.com/gluonhq/scenebuilder/pull/619/files#diff-fcff3bec73ef52f202e23cb21a1e0fd095b82bf66b6940fa5ed5b7165fc159bdL3

Reverting this change will work passing the same `-Djavafx.allowjs=true` to the surefire `<argLine/>`.

--- a/.github/scripts/jpackage.bat
+++ b/.github/scripts/jpackage.bat
@@ -23,6 +23,7 @@ REM set MODULES=java.desktop,java.logging,java.naming,java.prefs,java.security.j
 --runtime-image app/target/runtime ^
 --dest %INSTALL_DIR% ^
 --type msi ^
+--java-options "-Djavafx.allowjs=true" ^
 --java-options "--add-opens=javafx.fxml/javafx.fxml=ALL-UNNAMED" ^
 --java-options "-Djava.library.path=runtime\bin;runtime\lib" ^
 --icon app/assets/windows/icon-windows.ico ^

--- a/.github/workflows/bundles-linux.yml
+++ b/.github/workflows/bundles-linux.yml
@@ -77,6 +77,7 @@ jobs:
           .github/scripts/jpackage.sh \
             --icon app/assets/linux/icon-linux.png \
             --java-options '"-Djdk.gtk.version=2"' \
+            --java-options '"-Djavafx.allowjs=true"' \
             --java-options '"--add-opens=javafx.fxml/javafx.fxml=ALL-UNNAMED"' \
             --java-options '"-Djava.library.path=/opt/scenebuilder/lib/runtime/bin:/opt/scenebuilder/lib/runtime/lib"' \
             --linux-menu-group '"Development;Building;GUIDesigner;Java;"' \

--- a/.github/workflows/bundles-mac.yml
+++ b/.github/workflows/bundles-mac.yml
@@ -88,6 +88,7 @@ jobs:
         run: |
           .github/scripts/jpackage.sh \
             --icon app/assets/osx/icon-mac.icns \
+            --java-options '"-Djavafx.allowjs=true"' \
             --java-options '"--add-opens=javafx.fxml/javafx.fxml=ALL-UNNAMED"' \
             --type dmg \
             --mac-package-identifier com.gluonhq.scenebuilder \

--- a/.github/workflows/bundles-mac_aarch64.yml
+++ b/.github/workflows/bundles-mac_aarch64.yml
@@ -89,6 +89,7 @@ jobs:
         run: |
           .github/scripts/jpackage.sh \
             --icon app/assets/osx/icon-mac.icns \
+            --java-options '"-Djavafx.allowjs=true"' \
             --java-options '"--add-opens=javafx.fxml/javafx.fxml=ALL-UNNAMED"' \
             --type dmg \
             --mac-package-identifier com.gluonhq.scenebuilder \

--- a/kit/src/test/resources/com/oracle/javafx/scenebuilder/kit/fxom/HeaderWithNotOnlyImports.fxml
+++ b/kit/src/test/resources/com/oracle/javafx/scenebuilder/kit/fxom/HeaderWithNotOnlyImports.fxml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<?language nameOfScriptLanguage?>
+<?language javascript?>
 
 <!-- This could be a license header -->
 

--- a/pom.xml
+++ b/pom.xml
@@ -80,6 +80,7 @@
                     <configuration>
                         <mainClass>${main.class.name}</mainClass>
                         <options>
+                            <option>-Djavafx.allowjs=true</option>
                             <option>--add-opens=javafx.fxml/javafx.fxml=com.gluonhq.scenebuilder.app</option>
                             <option>--add-opens=javafx.fxml/javafx.fxml=com.gluonhq.scenebuilder.kit</option>
                             <option>--add-opens=javafx.controls/com.sun.javafx.scene.control.skin.caspian=com.gluonhq.scenebuilder.kit</option>
@@ -139,7 +140,7 @@
                     <configuration>
                         <reuseForks>false</reuseForks>
                         <forkCount>1</forkCount>
-                        <argLine>--add-opens=javafx.fxml/javafx.fxml=com.gluonhq.scenebuilder.kit</argLine>
+                        <argLine>--add-opens=javafx.fxml/javafx.fxml=com.gluonhq.scenebuilder.kit -Djavafx.allowjs=true</argLine>
                     </configuration>
                 </plugin>
             </plugins>

